### DISCUSSION TITLE
How to build?

### DISCUSSION THREAD

**Discussion by DreepyYunky** [dail8859/NotepadNext#478] [General]
Just having trouble with Qt Creator, I want a highly detailed description of how to build

**Comment by dail8859** [comment]
The documentation for building it is located here https://github.com/dail8859/NotepadNext/blob/master/doc/Building.md incase you haven't found it yet. That's the best there is so far. It is not perfect and I'm sure it can be improved since I created it after the fact.

Is there any kind of specific issues you are running into? I won't be able to help out too much for Linux and macOS.

**Reply by DreepyYunky** [reply]
Yeah I have, what do you mean by configure the project?

**Reply by dail8859** [reply]
When you first start QtCreator for a project it asks you what kits (e.g. versions of Qt, compiler, etc) to set up.

1. https://doc.qt.io/qtcreator/creator-configuring-projects.html
2. https://doc.qt.io/qtcreator/creator-project-opening.html

I'm not at my development machine currently but if/when I get a chance I can try to grab some screenshots if the above documentation isn't helpful.

**Reply by DreepyYunky** [reply]
Thanks, checking docs now
